symbolic-syntax/README.md

```diff
@@ -0,0 +1,40 @@
+# macaw-symbolic-syntax
+
+This package provides concrete syntax for macaw-symbolic types and operations.
+
+Concretely, it implements a `ParserHooks` for use with [`crucible-syntax`][syn].
+This `ParserHooks` supports the following types and operations:
+
+**Types**:
+
+The main type addition is for representing pointers:
+
+- `Pointer`
```

Comment:
> (Pointer just chooses the width for you depending on your architecture.)

This is an interesting point - as with the type aliases (e.g., `Long`), it's nice to have facilities for writing architecture-neutral syntax. However, this might be better served by making the parsers extensible, and introducing an API via which things like type aliases and architecture-width pointer types could be added, on top of the "basic" functionality of fixed-width pointers.

I think that since the two syntaxes have the same denotation (Crucible-LLVM's pointer type), it would make sense for them to be the same. I can expose the `(Ptr n)` type upstream, and see what I can do about making the parser more extensible. I'll try porting the type alias code here as an extension there.

Reply:
The parser now supports `(Ptr n)` by importing the parser from crucible-llvm-syntax, and also `Pointer`, which was upstreamed to crucible-llvm-syntax.